### PR TITLE
Fix the table for UD0 and UD1 (X86)

### PR DESCRIPTION
**Pull request by junghee**

### CODE REVIEW

File: arch/X86/X86MappingInsn.inc [hunk 1]
```diff
@@ -732,7 +732,7 @@
 },
 
 {
-	X86_ADD_FPrST0, X86_INS_FADD, 0,
+	X86_ADD_FPrST0, X86_INS_FADDP, 0,
```
Comment (aeflores) [inline arch/X86/X86MappingInsn.inc:735]: why this change? it seems unrelated to UD0 and UD1

**Review by aeflores — CHANGES_REQUESTED** [review]

File: arch/X86/X86MappingInsn.inc (hunk 1 above)
Comment (junghee) [reply arch/X86/X86MappingInsn.inc:735]: I don't really know why that is different. That file is generated. In fact, there is faddp in one of the X86 td files: capstone/suite/synctools/tablegen/X86/X86InstrFPStack.td:def ADD_FPrST0  : FPrST0PInst<MRM0r, "faddp\t$op">;

Comment (junghee) [reply arch/X86/X86MappingInsn.inc:735]: BTW, FYI, I plan to make more updates on this tomorrow.

File: arch/X86/X86MappingReg.inc [hunk 2]
```diff
@@ -110,14 +110,14 @@
...
-{ X86_K6, X86_REG_K6 },
-{ X86_K7, X86_REG_K7 },
+{ X86_K0, 0 },
+{ X86_K1, 0 },
+{ X86_K2, 0 },
+{ X86_K3, 0 },
+{ X86_K4, 0 },
+{ X86_K5, 0 },
+{ X86_K6, 0 },
+{ X86_K7, 0 },
```
Comment (aeflores) [inline arch/X86/X86MappingReg.inc]: this is something that I fixed a while ago https://github.com/capstone-engine/capstone/pull/1689
We should be careful, some of these differences that we see might be other fixes that people did after the table generation.

Comment (junghee) [reply arch/X86/X86MappingReg.inc]: That's right. I noticed that. I just pushed a fix. 
I do remember I had to edit those, but I don't really remember anything else.

File: arch/X86/X86GenRegisterInfo.inc [hunk 3]
```diff
@@ -382,10 +383,11 @@ enum {
   X86_VR512_with_sub_xmm_in_VR128HRegClassID = 84,
   X86_VR512_with_sub_xmm_in_VR128LRegClassID = 85,
 };
+
 #endif // GET_REGINFO_ENUM
 
 #ifdef GET_REGINFO_MC_DESC
-#define GET_REGINFO_MC_DESC
+#undef GET_REGINFO_MC_DESC
```
Comment (aeflores) [inline arch/X86/X86GenRegisterInfo.inc]: check git blame to see if this was fixings something

File: arch/X86/X86MappingReg.inc [hunk 4]
```diff
@@ -66,10 +66,10 @@
...
 { X86_SS, X86_REG_SS },
 { X86_SSP, 0 },
-{ X86_BND0, X86_REG_BND0 },
-{ X86_BND1, X86_REG_BND1 },
-{ X86_BND2, X86_REG_BND2 },
-{ X86_BND3, X86_REG_BND3 },
+{ X86_BND0, 0 },
+{ X86_BND1, 0 },
+{ X86_BND2, 0 },
+{ X86_BND3, 0 },
```
Comment (aeflores) [inline arch/X86/X86MappingReg.inc]: we probably want to keep the original changes here.

File: suite/synctools/insn.py [hunk 5]
```diff
@@ -112,7 +112,7 @@ def extract_matcher(filename):
 ##endif
 #},
 def print_entry(arch, insn_id, mnem, mapping, mnem_can_be_wrong):
-    print(arch, insn_id, mnem, mnem_can_be_wrong)
+    #print(arch, insn_id, mnem, mnem_can_be_wrong)
```
Comment (aeflores) [inline suite/synctools/insn.py:116]: add comment here

File: arch/X86/X86MappingInsn.inc (hunk 1 above)
Comment (junghee) [reply arch/X86/X86MappingInsn.inc:735]: I couldn't find anything about faddp in the git log history: i.e., there's no commit mentioning faddp.

File: arch/X86/X86GenRegisterInfo.inc (hunk 3 above)
Comment (junghee) [reply arch/X86/X86GenRegisterInfo.inc]: I found specific commits regarding GET_REGINFO_MC_DESC
```
0b690387b (Nguyen Anh Quynh  2014-08-13 13:01:50 +0800  387) #ifdef GET_REGINFO_MC_DESC
5a9962407 (Nguyen Anh Quynh  2019-02-26 15:19:51 +0800  388) #define GET_REGINFO_MC_DESC
0b690387b (Nguyen Anh Quynh  2014-08-13 13:01:50 +0800 1549) #endif // GET_REGINFO_MC_DESC
```
It looks like the second line is hand-edited.
So, I will make the change by hand.
